xml/System.Windows.Forms/FolderBrowserDialog.xml

```diff
@@ -479,10 +479,11 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a value that indicates whether to use the value of the <see cref="P:System.Windows.Forms.FolderBrowserDialog.Description" /> property as the dialog title for Vista style dialogs. This property has no effect on old style dialogs.</summary>
```

Comment:
This description is not very helpful, given that what "Vista style" means is unclear and "old style" (presumably pre-Vista) is also unclear.

Reply:
@RussKie @dreddy-work @Tanya-Solyanik  would you mind helping clarify and reword "Vista style"?

Reply:
This came as-is from the /// comments so a suggestion here would be good

Reply:
Good question! we don't support Vista or pre-Vista on Core 3.0.... 
@RussKie  - I think the right thing to do would be experiment how this works on "post vista" OSs and if there is no way to display the XP dialog, then to remove the property( mark it as obsolete or throw PlatformNotSupportedException). I doubt this makes the 3.0 bar (maybe as a removal of .NET-isms?) 

@carlossanlop - I'm not sure how to describe the dialog, as far as I remember it has buttons for folders in the user profile on the left, while the XP dialog does not... Let us experiment with these properties first, maybe they don't do anything on the newer OSs.

https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md

OS | Version | Architectures | Notes
-- | -- | -- | --
Windows Client | 7 SP1+, 8.1 | x64, x86 |  
Windows 10 Client | Version 1607+ | x64, x86 |  
Nano Server | Version 1803+ | x64, ARM32 |  
Windows Server | 2012 R2 SP1+ | x64, x86 |  

Reply:
>
> Let us experiment with these properties first, maybe they don't do anything on the newer OSs.

Ok sure. Please let us know what the results are.

One request, though: The deadline for merging this PR is tomorrow Thursday. We need to have everything merged on Friday so we can generate the IntelliSense nuget package out of this documentation.

Reply:
Given the deadline, we have no capacity to experiment nor capacity to change the API.

Can we leave it as-is, and then revisit if post the 3.0 release?

Reply:
Sure, here is the bug: https://github.com/dotnet/winforms/issues/1643
@carlossanlop we have to keep "Vista-style" until we can investigate further unfortunately.

Reply:
@Tanya-Solyanik are you saying you want to keep the current summary text as-is?

Reply:
Unfortunately, yes. We don't have the bandwidth to fix it properly right now.